Automatic Validation ended with:
> 2024-01-15T14:11:15.0122970Z ##[error] Manifest Error: Schema Error. [Schema validation failed. Error context: <root>[Tags] Description: Array should contain no more than 16 elements. Error context: <root> Description: Failed to validate against schema associated with property name 'Tags'. ] File: AderitoNeto.AfterLife.locale.en-US.yaml

(Automated response - build 570.)

--- a/manifests/a/AderitoNeto/AfterLife/1.0.0/AderitoNeto.AfterLife.locale.en-US.yaml
+++ b/manifests/a/AderitoNeto/AfterLife/1.0.0/AderitoNeto.AfterLife.locale.en-US.yaml
@@ -17,8 +17,6 @@ ReleaseNotesUrl: https://github.com/ad3rito/AfterLife/releases/tag/1.0.0
 ManifestType: defaultLocale
 ManifestVersion: 1.5.0
 Tags:
-- after
-- css
 - desktop-app
 - electron
 - electron-app
@@ -26,11 +24,5 @@ Tags:
 - html
 - javascript
 - life
-- linux
-- linux-app
-- macos
-- macos-app
 - rocket
 - space
-- windows
-- windows-app

